web/app/css/service-mesh.css

```diff
@@ -1,8 +1,6 @@
 @import 'styles.css';
 
 .header-with-metric {
```

Comment:
I can't figure out why but it seems as if the namespace table at the bottom of the service mesh page isn't at 100% width with its container. I tried messing with the CSS but couldn't seem to get it to work.

Reply:
Looking a little bit closer, it seems like something is going on with antd's table rendering where it will not give auto widths for table elements. We may have to manually provide column widths but I am hoping that is a last resort. This is true for any metrics table present in the dashboard. So you should see this happening in the Overview page, and the Resource Detail page.